x-pack/plugins/security_solution/common/search_strategy/security_solution/hosts/common/index.ts

```diff
@@ -68,7 +68,12 @@ export interface HostOsHitsItem {
     total: TotalValue | number;
     max_score: number | null;
     hits: Array<{
-      _source: { host: { os: Maybe<OsEcs> } };
+      fields: {
```

Comment:
Would like to know how do we decide which fields go here, as we used to put all the OsEcs fields here

Reply:
Yes, it's a good point. I'm suggesting to get back to the previous approach with having ECS type validation, but change correspondently the name of the property to `unflattenedFields` + provide the proper transformation of the parsing response from ES. @angorayc are you OK with the changes?